server/block/respawn_anchor.go

```diff
@@ -0,0 +1,96 @@
+package block
+
+import (
+	"github.com/df-mc/dragonfly/server/block/cube"
+	"github.com/df-mc/dragonfly/server/item"
+	"github.com/df-mc/dragonfly/server/player/chat"
+	"github.com/df-mc/dragonfly/server/world"
+	"github.com/df-mc/dragonfly/server/world/sound"
+)
+
+// RespawnAnchor is a block that allows the player to set their spawn point in the Nether.
+type RespawnAnchor struct {
+	solid
+	bassDrum
+	Charge int
+}
+
+// LightEmissionLevel ...
+func (r RespawnAnchor) LightEmissionLevel() uint8 {
+	return uint8(max(0, 3+4*(r.Charge-1)))
+}
+
+// EncodeItem ...
+func (r RespawnAnchor) EncodeItem() (name string, meta int16) {
+	return "minecraft:respawn_anchor", int16(r.Charge)
```

Comment:
```suggestion
	return "minecraft:respawn_anchor", 0
```

Only meta=0 is registered